core/test/financial-templates/Liquidatable.js

```diff
@@ -724,14 +724,18 @@ contract("Liquidatable", function(accounts) {
           const deletedLiquidation = await liquidationContract.liquidations(sponsor, liquidationParams.uuid);
           assert.equal(deletedLiquidation.liquidator, zeroAddress);
         });
-        it("Event emmited", async () => {
+        it("Event emitted", async () => {
           const withdrawalResult = await liquidationContract.withdrawLiquidation(liquidationParams.uuid, sponsor, {
             from: sponsor
           });
 
-          // TODO: flesh out this test with other params when they are added to the event in `Liquidatable.sol`
+          const expectedPayment = amountOfCollateral.sub(settlementTRV).add(sponsorDisputeReward);
           truffleAssert.eventEmitted(withdrawalResult, "LiquidationWithdrawn", ev => {
-            return ev.caller == sponsor;
+            return (
+              ev.caller == sponsor &&
+              ev.withdrawalAmount.toString() == expectedPayment.toString() &&
+              ev.liquidationStatus.toString() == STATES.DISPUTE_SUCCEEDED
```

Comment:
Should you add tests for when the dispute fails and or when liquidation liveness expires absent disputes?

Reply:
I can, sure.